test/e2e/EsLint.spec.ts

```diff
@@ -59,7 +60,7 @@ describe('EsLint', () => {
     errors = await driver.waitForErrors();
     expect(errors).toEqual([
       [
-        'WARNING in src/authenticate.ts 14:34-37',
+        fileLocation('WARNING in src/authenticate.ts', '14:34', async),
```

Comment:
I don't think we should have different format depending on the async mode. Could you try to modify WebpackError.ts file to make it consistent?

Reply:
Aaah, `IssueWebpackError.ts`, I missed that. Much better!